README.md

```diff
@@ -1 +1,23 @@
-# ecb_rates
+# Usage:
```

Comment:
I think I would be helpful to add information about source of exchange rates like website url.